ada_feeding/ada_feeding/visitors/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+This package contains custom BT Visitors that are used in the Ada Feeding
+project. All of these extend VisitorBase.
+"""
+
+from .moveit2_visitor import MoveIt2Visitor
```

Comment:
I think this should be called `MoveToVisitor`, since it is specifically for all the MoveTo trees, right?

Reply:
I agree specifically because it is for the MoveTo-style action interface.